tests/unit/hive_metastore/test_pipeline_migrate.py

```diff
@@ -65,7 +67,7 @@ def test_migrate_pipelines(ws, mock_installation, pipeline_spec, expected, api_c
     pipelines_crawler = PipelinesCrawler(ws, sql_backend, "inventory_database")
     jobs_crawler = JobsCrawler(ws, sql_backend, "inventory_database")
     pipelines_migrator = PipelinesMigrator(
-        ws, pipelines_crawler, jobs_crawler, "catalog_name", skip_pipeline_ids=["skip-pipeline"]
+        ws, pipelines_crawler, jobs_crawler, "catalog_name", include_pipeline_ids=[include_flag]
     )
 
     ws.jobs.get.side_effect = [
```

Comment:
Unrelated to these changes, below there are some mock return value and side effect after the asserts. Is that correct? They look redundant